packages/core/src/modules/event/event-repository.ts

```diff
@@ -122,4 +123,8 @@ export class EventRepositoryImpl implements EventRepository {
   async removeEventFromInterestGroup(eventId: EventId, interestGroupId: InterestGroupId): Promise<void> {
     await this.db.eventInterestGroup.delete({ where: { eventId_interestGroupId: { interestGroupId, eventId } } })
   }
+
+  async delete(id: EventId): Promise<void> {
+    await this.db.event.delete({ where: { id } })
+  }
```

Comment:
We can get to this later because it requires more code changes, but we should ideally never truly delete data.

There should probably be a `deleted: DateTime?` field on the event object, where `null` signals its active, and a time value tells the time it was "deleted"